mfa/methods/code/verify.md

```diff
@@ -0,0 +1,156 @@
+{% method %}
+
+## Two-Factor Verify Code
+Two-Factor authentication with Bandwidth services
+
+### Request URL
+
+<code class="post">POST</code>`https://mfa.bandwidth.com/api/v1/accounts/{accountId}/code/verify`
+
+#### Basic Authentication
+
+Bandwidth's Voice API leverages Basic Authentication with your Dashboard API Credentials. Read more about how Bandwidth secures endpoints in the [Security & Credentials](../../../guides/accountCredentials.md) document.
+
+---
+
+### Supported Parameters
+
+| Parameter     | Description                |
+|:--------------|:---------------------------|
+| to            | To telephone number        |
+| from          | From telephone number      |
+| applicationId | Application Id             |
```

Comment:
overly indicate that the applicationid should be the same as it was sent originally

Reply:
I like it, I changed it to `The voice or messaging application id used to make the initial request`